src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -282,11 +282,19 @@ The most convenient and cheap way of running Ethereum node is to use a single bo
 
 Small, affordable and efficient devices like these are ideal for running a node at home.
 
-## Eth2 clients {#eth2-clients}
+## Consensus clients {#consensus-clients}
```

Comment:
```suggestion
## Consensus clients (Eth2 clients)  {#consensus-clients}
```
I think for now (not sure for how long), we should explain these were previously referred to as Eth2 clients.

Reply:
Maybe we should also have `## Execution clients (Eth1 clients)`?